src/utils/helpers.ts

```diff
@@ -56,32 +56,25 @@ const helpers = {
    * @param {number} value Amount to be formatted
    *
    * @return {string} Formatted value
+   * @deprecated Import this function directly from `utils/numbers.ts`
    *
    * @memberof Helpers
    * @inner
    */
-  prettyValue(value: number): string {
-    const fixedPlaces = (value/10**DECIMAL_PLACES).toFixed(DECIMAL_PLACES);
-    const integerPart = fixedPlaces.split('.')[0];
-    const decimalPart = fixedPlaces.split('.')[1];
-    return `${this.prettyIntegerValue(parseInt(integerPart))}.${decimalPart}`;
-  },
+  prettyValue: prettyValue,
```

Comment:
Why do you need to keep this?

Reply:
For backward compatibility. Lib clients may be using it and now will have a warning because this method now have the `@deprecated` annotation on its docstrings.

After some time we can completely remove this property.

Reply:
We are about to release v1.0.0 that breaks compatibility with almost everything. I think this is the perfect momento to remove this extra code and just document in the release notes that methods A, B, and C were moved to file X, and methods D and E were moved to file Y. What do you think?

Reply:
Good point! I used to opportunity to also refactor the tests into their own files.